WishList/WishList/Login/SignInView.swift

```diff
@@ -88,11 +88,60 @@ struct SignInView: View {
             .padding()
         }
     }
+    private func createUser() {
```

Comment:
Change this method name. It doesn't create an user, it is login in the user.

Reply:
Also, it is not being called at all. This should be called when the user press the sign in button.